library/src/scala/tasty/Reflection.scala

```diff
@@ -2409,21 +2409,25 @@ trait Reflection { reflection =>
   // IMPLICIT SEARCH //
   /////////////////////
 
-  // TODO: this should not be top level
-  /** Find an implicit of type `T` in the current scope given by `ctx`.
-   *  Return an `ImplicitSearchResult`.
-   *
-   *  @param tpe type of the implicit parameter
-   *  @param ctx current context
-   */
-  def searchImplicit(tpe: Type): ImplicitSearchResult
+  /** Result of an implicit search */
+  type ImplicitSearch <: AnyRef
+
+  given TypeTest[ImplicitSearch, ImplicitSearchSuccess] = ImplicitSearchSuccessTypeTest
+  protected val ImplicitSearchSuccessTypeTest: TypeTest[ImplicitSearch, ImplicitSearchSuccess]
 
-  type ImplicitSearchResult <: AnyRef
+  val ImplicitSearch: ImplicitSearchModule
```

Comment:
Do we still want to call this an “implicit” search? Or a “contextual” search? Or a “given instance” search?

Reply:
Last time I proposed that somewhere opposed. I would be up for the change. Which one should we use?

Reply:
I’m fine with “implicit” but I wanted to double-check because there are no mentions of “implicit” anymore in the Dotty documentation.